backend/handlers/handlers.go

```diff
@@ -193,6 +195,8 @@ func NewHandlers(
 	getAPIRouterNoError(apiRouter)("/update", handlers.getUpdateHandler).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/banners/{key}", handlers.getBannersHandler).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/using-mobile-data", handlers.getUsingMobileDataHandler).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/authenticate", handlers.getAuthenticateHandler).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/trigger-auth", handlers.getTriggerAuthHandler).Methods("GET")
```

Comment:
these feel more like they should be POST - GET is for retrieving data, POST is to change state.